client/my-sites/invites/p2/invite-accept-logged-out.jsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import SignupForm from 'calypso/blocks/signup-form';
+import { renderInviteAcceptP2Logo } from './invite-accept-logo-line';
+import { renderInviteAcceptFormHeader } from './invite-accept-form-header';
+import { renderInviteAcceptFooter } from './invite-accept-footer';
+import LoggedOutFormLinks from 'calypso/components/logged-out-form/links';
+import LoggedOutFormLinkItem from 'calypso/components/logged-out-form/link-item';
+
+function renderFooterLink( props ) {
+	return (
+		<div className="invite-accept-logged-out__footer-link">
+			<div>{ props.translate( 'Already have a WordPress.com account? ' ) }</div>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **45** times:
`translate( 'Already have a WordPress.com account?' )` *ES Score: 7*